src/Behat/Context/MediaContext.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+use Metadrop\Behat\Context\RawDrupalContext;
+use Metadrop\Behat\Context\UIContext;
+use Metadrop\Behat\Context\WaitingContext;
+use Behat\Gherkin\Node\TableNode;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+
+/**
+ * Steps related with media.
+ */
+class MediaContext extends RawDrupalContext {
+
+  protected $medias = [];
```

Comment:
This property can be removed, same reasons as https://github.com/Metadrop/behat-contexts/pull/131/files#r838189726